src/components/icon/graphics/ebplus-white.svg

```diff
@@ -0,0 +1,11 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 365 365">
```

Comment:
Nu hvor der er mulighed for hvide graphics, så kan den ikke ses på nuværende hvide baggrund. Kunne være vi skulle lave baggrunden bg-graa5 under _icons_